client/commands.c

```diff
@@ -446,7 +446,8 @@ void processStatsCommand(FILE *out)
 			" wait sync blocks: %u\n"
 			" chain difficulty: %llx%016llx of %llx%016llx\n"
 			" %9s supply: %.9Lf of %.9Lf\n"
-			"4 hr hashrate MHs: %.2Lf of %.2Lf\n",
+			"4 hr hashrate MHs: %.2Lf of %.2Lf\n"
+			"    cached blocks: target amount %u, actual amount %u, hitrate %f%%\n",
```

Comment:
Information about cached blocks is more internal technical thing. I think it makes sense to remove it after testing before release.

Reply:
i think it is needed, we can add a command to see cache infos.